src/client/components/TablePaginated/hooks/useFilterValues.tsx

```diff
@@ -0,0 +1,35 @@
+import { useMemo } from 'react'
+
+import { TablePaginatedFilterType, TablePaginatedFilterValues } from 'meta/tablePaginated'
+
+import { useTablePaginatedFilters } from 'client/store/ui/tablePaginated/hooks'
+import { TablePaginatedFilter } from 'client/components/TablePaginated/types'
+
+type Props = {
+  filters: Array<TablePaginatedFilter<TablePaginatedFilterType>>
+  path: string
+}
+
+type Returned = Record<string, TablePaginatedFilterValues>
+
+export const useFilterValues = (props: Props): Returned => {
+  const { filters, path } = props
+
+  const activeFilterValues = useTablePaginatedFilters(path)
+
+  const hiddenFilterValues = useMemo<Returned>(() => {
+    return filters.reduce((acc, { hidden, defaultValue, fieldName }) => {
+      if (hidden && defaultValue !== undefined) {
+        // eslint-disable-next-line no-param-reassign
+        acc[fieldName] = defaultValue
+      }
+      return acc
+    }, {} as Returned)
+  }, [filters])
+
+  const allFilterValues = useMemo<Returned>(() => {
+    return { ...hiddenFilterValues, ...activeFilterValues }
+  }, [activeFilterValues, hiddenFilterValues])
+
+  return allFilterValues
+}
```

Comment:
The last thing missing would be to handle filters that are not hidden and have a default value (like having a prefilled name for the text input, for instance). However, that's not necessary for this issue.